tests/integration/test_numpy1.pyi

```diff
@@ -1,12 +1,48 @@
-from typing import Any
+# mypy: disable-error-code="no-redef"
 
-# requires numpy < 2
-import numpy.array_api as np
+from types import ModuleType
+from typing import Any, TypeAlias
+
+import numpy.array_api as np  # type: ignore[import-not-found, unused-ignore]
 
 import array_api_typing as xpt
 
-###
-# Ensure that `np.ndarray` instances are assignable to `xpt.HasArrayNamespace`.
+# DType aliases
+F32: TypeAlias = np.float32
+I32: TypeAlias = np.int32
+
+# Define NDArrays against which we can test the protocols
+nparr = np.eye(2)
+nparr_i32 = np.array([1], dtype=I32)
+nparr_f32 = np.array([1.0], dtype=F32)
+nparr_b = np.array([True], dtype=np.bool_)
```

Comment:
yea, it's too bad we can't properly annotate this...

Reply:
Yeah. But thankfully np<2 is not long for this world so it's a transient annoyance.

Reply:
Spoke too soon. It's a real pain.

Reply:
😆